api/app/serializers/maker_serializer.rb

```diff
@@ -1,8 +1,4 @@
 # Provides a partial serialization of a maker model.
-class MakerSerializer < ActiveModel::Serializer
-  meta(partial: false)
-
-  cache key: "maker", expires_in: 3.hours
-  attributes :id, :first_name, :last_name, :middle_name, :display_name, :avatar_url,
-             :full_name
+class MakerSerializer < MakerPartialSerializer
+  has_many :users, serializer: UserPartialSerializer
```

Comment:
Given that we never want to expose the user on the maker, I think we can go ahead and collapse the MakerSerializer and the MakerPartialSerializer back down to a single serializer. I assume you did this just to break out the include, which shouldn't be necessary.